module/actor/actor.js

```diff
@@ -173,6 +168,15 @@ export class yzecoriolisActor extends Actor {
     return bonus;
   }
 
+  migrateKeyArt() {
+    if (
+      (this.type === "npc" || this.type === "character") &&
+      this.system.keyArt
+    ) {
+      this.update({ img: this.system.keyArt, system: { keyArt: "" } });
```

Comment:
I initially wanted to remove the keyArt field from template.json, but that could potentially break things since Foundry might remove the field when using the newest version of the system and then we have no way of knowing what the character art should be.

Reply:
It's been over a year since I've done a data migration on Coriolis. I'll dig into this to see if I can just update the template and do the migration like I have been. My suspicion is with V10+ it culls any unknown fields rather than letting them through.